bin/servicemon.py

```diff
@@ -27,27 +27,36 @@
 import threading
 import signal
 import argparse
-from nav import buildconf
+import logging
 
+from nav import buildconf
 import nav.daemon
 from nav.daemon import safesleep as sleep
-from nav.statemon import RunQueue, config, db, debug
+from nav.logs import init_generic_logging, convert_debug_level_to_loglevel
+from nav.statemon import RunQueue, config, db
+
+
+LOGGER = logging.getLogger('bin.servicemon')
 
 
 class Controller:
     def __init__(self, **kwargs):
         signal.signal(signal.SIGHUP, self.signalhandler)
         signal.signal(signal.SIGTERM, self.signalhandler)
         self.conf = config.serviceconf()
-        debug.setDebugLevel(int(self.conf.get('debuglevel', 4)))
+        debuglevel = int(self.conf.get('debuglevel', 4))
+        init_generic_logging(
+            stderr=True, read_config=True,
+            stderr_level=convert_debug_level_to_loglevel(debuglevel)
+        )
         self._deamon = kwargs.get("fork", 1)
         self._isrunning = 1
         self._checkers = []
         self._looptime = int(self.conf.get("checkinterval", 60))
-        debug.debug("Setting checkinterval=%i" % self._looptime)
+        LOGGER.warning("Setting checkinterval=%i", self._looptime)
```

Comment:
The default for `servicemon` is 4, not 5, scroll up a little: 

`debug.setDebugLevel(int(self.conf.get('debuglevel', 4)))`

This is the only time it is overridden to 4.